src/unless.js

```diff
@@ -21,10 +21,9 @@ var _curry3 = require('./internal/_curry3');
  * @see R.ifElse, R.when
  * @example
  *
- *      // coerceArray :: (a|[a]) -> [a]
- *      var coerceArray = R.unless(R.isArrayLike, R.of);
- *      coerceArray([1, 2, 3]); //=> [1, 2, 3]
- *      coerceArray(1);         //=> [1]
+ *      let longUpper = R.unless(x => x.length < 3, R.toUpper);
```

Comment:
This isn't a great example for `R.unless`, as it could easily be expressed in terms of `R.when`.